lua/entities/gmod_wire_rt_camera.lua

```diff
@@ -66,12 +70,14 @@ if CLIENT then
 
     local function SetCameraActive(camera, isActive)
         if isActive then
-            ActiveCameras[camera] = true
+            if not table.HasValue(ActiveCameras, camera) then
+                table.insert(ActiveCameras, camera)
```

Comment:
Why not use a data structure that has both an array and LUT?

Reply:
Because it is never used as lookup table. Here `table.HasValue` is used to ensure camera not added into array twice.

Reply:
A lookup table should be faster though. It looks like you replaced a lookup table with a sequential table/array in multiple places which should only degrade performance for literally no gain. Is there a reason you did that?

Reply:
Here `ActiveCameras` is iterated each time cameras are rendered and modified when cameras' `Active` input is changed.
As camera rendering is more frequent than `Active` changing, I have chosen to optimise iteration.
Previously, `pairs` was used in loop, and now `ipairs` is.

Reply:
Oh okay, I only read this code segment so I was a little confused. I guess it's hard to say which would be faster overall but either way it shouldn't matter too much. Fair enough then.